Client/Frontend/Browser/Playlist/Managers & Cache/PlaylistCacheLoader.swift

```diff
@@ -394,7 +394,7 @@ class PlaylistWebLoader: UIView {
     }
 
     // TODO: REFACTOR to support multiple windows better
-    if let browserController = webView.currentScene?.browserViewController {
+    if let browserController = webView.currentScene?.browserViewController ?? UIApplication.shared.keyWindow?.windowScene?.browserViewController {
```

Comment:
There isn't any way to pass it in all the time. This is also done to fix a bug in CarPlay which doesn't require a `browserViewController` since it has its own app-delegate. So unless we refactor `profile` and `certStore` to be its own thing, that isn't always possible.

Reply:
I realize that but we really should not be reaching into UIApplication for _anything_ anymore. Any further changes that do this are just making future refactors more difficult

Reply:
Fixed